source/features/default-branch-button.tsx

```diff
@@ -55,7 +56,7 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.isRepoTree,
 		pageDetect.isSingleFile,
-		pageDetect.isRepoCommitList,
+		isRepoCommitListRoot,
```

Comment:
"List commits for files/folders" don't have a brach selector, so we don't add a default branch button either.

Compare:

- https://github.com/refined-github/sandbox/commits/4679
- https://github.com/refined-github/sandbox/commits/4679/README.md